src/dhdtOverrideManager.cpp

```
@@ -64,52 +66,44 @@ std::string hdt::Override::OverrideManager::checkOverride(uint32_t actor_formID,
+	if (!checkPapyrusExtension()) {
 		return;
 	}
 
 	m_ActorPhysicsFileSwapList.clear();
-	try 
-	{
-		while (!data_stream.eof()) 
-		{
+	try {
+		while (!data_stream.eof()) {
 			uint32_t actor_formID, override_size = 0;
```

Clang (14.0.6)

[warning] variable 'actor_formID' is not initialized

(cppcoreguidelines-init-variables)